types/node/ts3.1/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+    "files": [
+        "index.d.ts",
+        "../common.d.ts",
+        "../inspector.d.ts",
+        "node-tests.ts"
+    ],
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "lib": [
+            "es6",
```

Comment:
Lib `dom` is included to ensure that node + dom is compatible but do we still need `es6` here?

Reply:
We need `es5` at a minimum.